components/TradingCardImage.jsx

```diff
@@ -0,0 +1,35 @@
+export default function TradingCardImage({ imageUrl, hours }) {
+  const getModifier = (availableHours) => {
+    if (availableHours === 0) return 'gray';
+    if (availableHours >= 16) return 'green';
+    return 'blue';
+  };
+
+  const modifier = getModifier(hours);
+
+  return (
+    <article className="cmp-trading-card-image">
```

Comment:
I think `article` makes more sense for the `<TradingCard>`, not necessarily this component. `div` ought to be fine.